Ktisis/Util/EnumFlagExtension.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Ktisis.Util
+{
+	static class EnumFlagExtension
+	{
+		public static bool FlagHas<T>(this System.Enum type, T value) => ((int)(object)type & (int)(object)value) == (int)(object)value;
```

Comment:
This already exists as an instance method on `System.Enum` -- namely `HasFlag`. The other methods here are fine, but this one in specific already has a native implementation.

If you could change your calls to that method and then remove this method here, that'd be great.